### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by deshack**

### CODE REVIEW

File: Lib/Commands/CommandInterface.php [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+   * @access public
+   */
+  public function ignoreValidationErrors();
+
+  /**
+   * Whether command is enabled or not in the current environment.
+   *
+   * This should return false if the command can not run properly under the
+   * current conditions.
+   *
```
Comment (wraxen) [inline Lib/Commands/CommandInterface.php:41]: Missing `@since` and `@access`.

File: Lib/FreyjaCLI.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  protected $logger;
+
+  /**
+   * Class constructor.
+   *
+   * @since 0.1.0
+   * @access public
+   *
+   * @param
+   */
```
Comment (wraxen) [inline Lib/FreyjaCLI.php:59]: Constructor was kidnapped by aliens D:

File: Lib/Input/InputInterface.php [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  /**
+   * Set argument value by name.
+   *
+   * @since 1.0.0
+   * @access public
+   *
+   * @param string $name Argument name.
+   * @param string $value Argument value.
+   *
+   * @throws InvalidArgumentException if given argument does not exist.
```
Comment (wraxen) [inline Lib/Input/InputInterface.php]: `\InvalidArgumentException` (missing backslash) or `Freyja\Exceptions\InvalidArgumentException`

File: Lib/Input/InputInterface.php [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  /**
+   * Set option value by name.
+   *
+   * @since 1.0.0
+   * @access public
+   *
+   * @param string $name Option name.
+   * @param string|bool $value Option value.
+   *
+   * @throws InvalidArgumentException if option given does not exist.
```
Comment (wraxen) [inline Lib/Input/InputInterface.php]: See [comment](https://github.com/SqueezyWeb/freyja-cli/pull/4/files#r51756174) on [line 117](https://github.com/SqueezyWeb/freyja-cli/pull/4/files#diff-0c5542f9248c46342d27b4a4964d53e7R117)

File: Lib/Input/Option.php [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+
+  /**
+   * Set default value.
+   *
+   * @since 1.0.0
+   * @access public
+   *
+   * @param mixed $default Default value.
+   *
+   * @throws LogicException if incorrect default value is given.
```
Comment (wraxen) [inline Lib/Input/Option.php]: Backslash before exception class: `\LogicException`

File: Lib/FreyjaCLI.php (hunk 2 above)
Comment (deshack) [inline Lib/FreyjaCLI.php:59]: We need to define internal logic first ;)

File: Lib/Commands/CommandInterface.php (hunk 1 above)
Comment (deshack) [inline Lib/Commands/CommandInterface.php:41]: Fixed with c106365

**Pull request closed without merging**